### PR TITLE
feat: ignore price database

### PR DESCRIPTION
**Pull request by MounirDhahri**
_I apologize for the big diff, I alphabetically sorted the paths to make it easier to add new ones in the future._

This PR resolves https://artsyproduct.atlassian.net/browse/CX-2113

**Description:**
The goal of this PR is to make sure that the price-database link does not open the app and instead opens the link in the browser

**Note:**
The next steps here would be updating artsy-eigen-web-association in artsy-wwwify and force

### CODE REVIEW

File: apple-app-site-association.json [hunk 1]
```diff
@@ -17,44 +17,44 @@
...
-          "NOT /click/*/aHR0cHM6Ly9hcnRzeS5uZXQv*",
-          "NOT /click/*/aHR0cHM6Ly9hcnRzeS50eXBlZm9ybS5j*",
-          "NOT /click/*/aHR0cHM6Ly9hcHAuYWRqdXN0LmNv*",
           "NOT /users/auth/facebook/callback*",
-          "NOT /identity-verification*",
-          "NOT /reset_password",
-          "NOT /change-password",
-          "NOT /forgot",
-          "NOT *L2FydGljbGVz*",
-          "NOT *L2FydGljbGUv*",
```
Comment (lordkiz) [inline apple-app-site-association.json:41]: @MounirDhahri This PR removes these two entries completely. Is that intended?

Comment (MounirDhahri) [reply apple-app-site-association.json:41]: I also noticed that. These two has been deleted after I ran `yarn build`

**Review by lordkiz — APPROVED** [review]
🚀 . Looks good. I think we can merge if the deleted ones are not a problem.

File: apple-app-site-association.json (hunk 1 above)
Comment (lordkiz) [reply apple-app-site-association.json:41]: Ok> Thanks for clarifying.

**Comment by artsyit** [comment]
:rocket: PR was released in [`v1.1.0`](https://github.com/artsy/artsy-eigen-web-association/releases/tag/v1.1.0) :rocket:

**Pull request merged**